addons/mail/static/src/owl/components/chatter/chatter.js

```diff
@@ -22,22 +22,43 @@ class Chatter extends Component {
             const chatter = state.chatters[props.chatterLocalId];
             const threadLocalId = chatter.threadLocalId;
             const thread = threadLocalId ? state.threads[threadLocalId] : undefined;
+            let attachments;
             if (thread) {
-                const attachments = thread.attachmentLocalIds.map(attachmentLocalId =>
+                attachments = thread.attachmentLocalIds.map(attachmentLocalId =>
```

Comment:
missing `compareDepth: { attachments: 1 }`

Reply:
As second argument of map ? I don't know that one .. ?

Reply:
this is an option of `useStore(selector, options)`, so `useStore(/* selector */, { compareDepth: { attachments: 1 } })` (ofc multi-line for scoring style points)

Reply:
Okay didn't know about that !